apps/webstore/src/app/shared/components/items/checkout-item/checkout-item.component.html

```diff
@@ -2,7 +2,7 @@
     <img src="/assets/img/design-ex1-img.png" class="item-image" />
     <div class="item-info">
         <div class="item-title">
-            <strong>{{ item.title }}</strong>
```

Comment:
Why change the title to quantity? This is the text that appears with the design img that is just supposed to show what design it is

Reply:
Because it was in the midst of applying a new interface and the location of the title has changed

Resolved in 015c398